### ISSUE TITLE
Fakegato Support

### ISSUE THREAD

**Issue by mike4president** [marcsowen/homebridge-homematicip#202]
Hi, any news on implementing Fakegato support?

**Comment by marcsowen** [comment]
Hi @mike4president, as you might have noticed I'm currently swamped with other projects and don't habe too much time working on the plugin. But I'm using it daily and will continue to update it incrementally. That being said, Fakegato support is on my Todo list and it will come eventually. But not in the next couple of weeks.

**Comment by mike4president** [comment]
Hi, I just noticed today. FAKEGATO SUPPORT. Thanks. But what do I have to do to activate it? It doesn't seem to work after updating the plugin. Ist there something I need to do to get I working?

**Comment by mike4president** [comment]
Forget what I jus wrote. I looked at the radiator. :-/ I just noticed it works perfectly as described for temperature and humidity sensors only. Thanks a lot. Are there more sensors planned for example door and windows sensors, light switches an so on?

**Comment by marcsowen** [comment]
@mike4president Sure, I will have a look at it.

**Comment by marcsowen** [comment]
@mike4president There are only two battery states provided by the HmIP-Cloud:

- Battery is full (LowBat = false)
- Battery is almost empty (LowBat = true)

HomeKit has a similar flag:

- BATTERY_LEVEL_NORMAL
- BATTERY_LEVEL_LOW

The HomematicIP plugin translates this information directly. Home app will show a battery almost empty warning when the accessory is in BATTERY_LEVEL_LOW state. The Eve app shows a full battery symbol for BATTERY_LEVEL_NORMAL and an almost empty symbol for BATTERY_LEVEL_LOW.

**Comment by mike4president** [comment]
Understood! Thanks for the fast reply. But why can’t I look up the reported battery state for all my devices using the „Typen“ feature? If I open the accessory directly I can see it but it ist not reported under the „Typen“ category 

![6D4D3CCD-A597-45EB-80B3-D213EDADB305](https://user-images.githubusercontent.com/65063329/143874769-1cd0842f-433a-42a1-8ccb-2e251c6bd2bb.jpeg)
![D61639CD-9E30-43D9-8B08-C9A2B2665147](https://user-images.githubusercontent.com/65063329/143874788-bdb2fc05-76c7-435a-b672-cf06b52b200e.jpeg)

**Comment by marcsowen** [comment]
@mike4president I can only guess: There is a second property for HomeKit where you can provide the battery level in percent. This probably qualifies as a real value for the Eve app.

The plugin used to generate a fake value (LowBat = false -> 100%, LowBat = true -> 10%) since it used to be a mandatory property for HomeKit. Now we only provide the information we actually have (LowBat true/false). It's up to the apps to do something useful with this.

**Comment by mike4president** [comment]
Makes sense. I tried the HA-Homemati Plugin for Raspberrymativ before and it worked as described in the Eve App. Couldn't you translate the values of LowBat = false to 100% and LowBat = true to 10% even if it is not true at all time to make it usable in the Eve-App?

**Comment by mike4president** [comment]
Hi Marc; still love this plugin and use it every day. If you wouldn’t have made this plugin I would have sold my Homematic IP system already to switch to something more open like Aqara. Any chance more devices get fakegato support anytime in the near future?

**Comment by marcsowen** [comment]
Fakegato history support was added in version 0.6.0 for temperature and humidity data. The original request is therefore resolved. History for switches and door/window sensors is not supported yet. If this is still relevant, can you open up a new issue with a feature request?
